runtime/lib/component/light.js

```diff
@@ -85,12 +87,26 @@ Light.prototype.appSound = function (appId, uri) {
  * @return {Promise}
  */
 Light.prototype.ttsSound = function (appId, uri) {
+  if (this.ttsSoundCnt[appId] == null) {
```

Comment:
Should we handle the case that isn't number or nan?

Reply:
It's not possible honestly speaking since it's on controlled scope. Though the case should be covered.